app/cdap/components/Cloud/Profiles/CreateView/index.js

```diff
@@ -182,6 +230,9 @@ class ProfileCreateView extends Component {
   };
 
   renderGroup = (group) => {
+    if (group.show === false) {
```

Comment:
Is this because you don't want undefined or null?

Reply:
Yes, in some cases it's undefined.